frontend/packages/client/src/networks.js

```diff
@@ -7,6 +7,11 @@ const networksConfig = {
     walletDiscoveryApi: null,
     walletDiscoveryInclude: [],
     strategiesConfig: {},
+    flowAddress: {
+      contractName: 'FlowToken',
+      contractAddr: '0x0ae53cb6e3f42a79',
+      storagePath: 'flowTokenVault',
```

Comment:
ah I checked this. It's now always `flowTokenBalance` for all environments. It changed a few times before.

Reply:
cool, updated to use flowTokenBalance in the config